### PR TITLE
Add the big shape test case for the op of matmul.

### PR DESCRIPTION
**Pull request by wawltor**
test=develop

### CODE REVIEW

**Comment by zhupengyang** [comment]
LGTM @luotao1

File: python/paddle/fluid/tests/unittests/test_matmul_op.py [hunk 1]
```diff
@@ -241,5 +241,22 @@ def generate_compatible_shapes(dim, transpose_X, transpose_Y):
                 'transpose_Y': transpose_Y,
             })
 
+
+class BigShapeTestCase(Generator):
+    def setUp(self):
+        self.op_type = "matmul"
+        self.transpose_X = False
+        self.transpose_Y = False
+        X = np.random.random([100, 103]).astype("float32")
```
Comment (luotao1) [inline python/paddle/fluid/tests/unittests/test_matmul_op.py]: @zhupengyang Is this `shape[100,103]` too large? And does this Op still not satisfy the `all cases use large shape` like in #22065?

**Comment by luotao1** [comment]
@wawltor Please make the PR description more details.

File: python/paddle/fluid/tests/unittests/test_matmul_op.py (hunk 1 above)
Comment (zhupengyang) [reply python/paddle/fluid/tests/unittests/test_matmul_op.py]: 确实还不满足现在shape规范升级后的要求；这么改主要是因为matmul的单测比较复杂，所以单独加了一个大shape的case，先满足之前的规范；个人认为可以先close掉，后面升级规范加完检查的时候再修复。 @luotao1

**Pull request closed without merging**